Source/WebCore/dom/RejectedPromiseTracker.h

```diff
@@ -59,7 +59,7 @@ class RejectedPromiseTracker {
     void reportUnhandledRejections(Vector<UnhandledPromise>&&);
     void reportRejectionHandled(Ref<DOMPromise>&&);
 
-    ScriptExecutionContext& m_context;
+    WeakPtr<ScriptExecutionContext> m_context;
```

Comment:
This is also a clear use case for `CheckedRef<ScriptExecutionContext>` since m_context is the owner and can never be null (cc @rniwa who wrote CheckedRef).

Reply:
Same comment may apply in other places in this patch.